SkiaChart/Charts/ChartBase.cs

```diff
@@ -207,16 +207,35 @@ public abstract class ChartBase : IChart {
             IsAntialias = true,
             Style = SKPaintStyle.Stroke,
             Color = SKColors.Green,
-            StrokeWidth = 5,
+			//StrokeWidth = 5,
             StrokeCap = SKStrokeCap.Round
         };
 
+		private float _labelTextSize = 10f;
+		/// <summary>
+		/// The TextSize of labels
+		/// </summary>
+		public float LabelTextSize
+		{
+			get => _labelTextSize;
+			set
+			{
+				if (value != _labelTextSize)
+				{
+					_labelTextSize = value;
+					_labelPaint.TextSize = value;
+				}
+			}
+		}
+
         //The SkPaint used for drawing labels
         protected readonly SKPaint _labelPaint = new SKPaint() {
             Style = SKPaintStyle.Stroke,
-            IsAntialias = true,
-            StrokeWidth = 3,
-            Color = SKColors.Gray
+			IsAntialias = true,
+			//StrokeWidth = 0.2f,
```

Comment:
Same here, I think we should remove those lines instead of commenting them out. It might start leaving the code base messing